frontend/containers/open-call-page/header/component.tsx

```diff
@@ -0,0 +1,190 @@
+import { FC, useMemo } from 'react';
+
+// import { Heart } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import { useRouter } from 'next/router';
+
+import dayjs from 'dayjs';
+
+import { translatedLanguageNameForLocale } from 'helpers/intl';
+
+import Breadcrumbs from 'containers/breadcrumbs';
+import ShareIcons from 'containers/share-icons';
+
+// import Button from 'components/button';
+// import Icon from 'components/icon';
+import LayoutContainer from 'components/layout-container';
+import { OpenCallStatus } from 'enums';
+
+import OpenCallChart from '../chart';
+
+import { OpenCallHeaderProps } from '.';
+
+export const OpenCallHeader: FC<OpenCallHeaderProps> = ({
+  openCall,
+  instrumentTypes,
+  // handleFavorite,
+  // handleApply,
+}) => {
+  const intl = useIntl();
+  const { locale } = useRouter();
+  const {
+    name,
+    instrument_types,
+    maximum_funding_per_project,
+    picture,
+    language,
+    created_at,
+    closing_at,
+    status,
+  } = openCall;
+  const coverImage = picture?.medium || '/images/avatar.svg';
```

Comment:
This fallback is wrong. If there are no cover image, we should show a green background.
```suggestion
  const coverImage = picture?.medium;
```